guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -0,0 +1,30 @@
+[id="disabling-cockpit-on-project_{context}"]
+= Disabling {CockpitTitle} on {Project}
+
+Perform the following procedure if you want to disable {Cockpit} on {Project}.
+
+.Procedure
+. Disable {Cockpit} with the `{foreman-installer}` command:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --no-enable-foreman-plugin-remote-execution-cockpit
+----
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Remote execution*.
+. Erase the value of the *Cockpit URL* setting.
+. Uninstall the {Cockpit} package from {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{package-remove} rubygem-foreman_remote_execution-cockpit
```

Comment:
The package name is different on Debian, but given we don't package it at all on Debian I'd hide the whole chapter on `foreman-deb`.

Reply:
Done :+1: